lib/generators/init-generator.js

```diff
@@ -340,7 +340,7 @@ module.exports = class InitGenerator extends Generator {
 				if (regExpForStyles) {
 					if (this.isProd) {
 						this.configuration.config.topScope.push(tooltip.cssPlugin());
-						this.dependencies.push("extract-text-webpack-plugin");
+						this.dependencies.push("extract-text-webpack-plugin@next");
```

Comment:
Could add a `TODO` here as a reminder up revert it once `v4.0.0` is out?

Reply:
I prefer such hard-coded strings in a variable rather hard-coded. Easier to remember, easier to change and yes for `TODO`

Reply:
I think this is pretty intuitive